oaknational/Oak-Web-Application
# feat: shareable new content - LESQ-915

## Description

Music year: 1959

- Activate share button on lesson overview page for new lessons, 

## Issue(s)

## How to test

1. Go to https://deploy-preview-2544--oak-web-application.netlify.thenational.academy
All post legacy lessons (new lessons) should have share all button on lesson overview page.

## Screenshots

How it used to look (delete if n/a):
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/da312edb-e67c-4abf-8a20-6d71db955b5b)

How it should now look:
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/3ff75309-f7bb-4271-bb6e-f5f79547974e)

## Checklist
